lib/PostMessageTransport.js

```diff
@@ -52,8 +52,8 @@ define(function (require, exports, module) {
         win.postMessage("initial message", "*");
 
         window.addEventListener("message", _listener);
-
-        exports.emitEvent("PostMessageTransport", "connect", [connId, "fakeURL"]);
+        //trigger connect event
+        module.exports.trigger("PostMessageTransport", "connect", [connId, "fakeURL"]);
```

Comment:
We should file a follow-up bug to get rid of the need to send the connection id and fakeurl stuff.  I bet we don't really need that if we're careful.  This is fine for now, though.